src/libData/MiningData/DSPowSolution.h

```diff
@@ -35,6 +35,8 @@ class DSPowSolution {
   std::string m_mixHash;
   uint32_t m_lookupId{};
   uint128_t m_gasPrice;
+  std::pair<uint32_t, uint32_t>
```

Comment:
Also, should we make this an atomic type? In case for example `storeVoteUntilPow` is called at the same time this value is cleared out (or other read/write race condition scenarios)

Reply:
Created.
`std::pair<std::atomic<uint32_t>, std::atomic<uint32_t>> m_govProposal;
  std::atomic<uint32_t> m_govMaxVoteAttempt{};`